src/shared/components/common/emoji-picker.tsx

```diff
@@ -0,0 +1,99 @@
+import { Component } from "inferno";
+import { i18n } from "../../i18next";
+import { getEmojiMart } from "../../utils";
+import { Icon } from "./icon";
+
+
+interface EmojiMartProps {
+    onEmojiClick?(val: any): any;
+    pickerOptions: any;
```

Comment:
Emoji-mart has typescript typings : https://www.npmjs.com/package/@types/emoji-mart

Reply:
The typings are over 3 years old, unfortunately, so they are basically useless. Ill try to define my own types.